app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,7 @@
     <%= stylesheet_pack_tag 'application', media: 'all' %>
     <%= javascript_pack_tag 'application', defer: true %>
     <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= yield(:javascript_pack_tags) %>
```

Comment:
Do we need this? 

I would think `<%= javascript_pack_tag 'application', defer: true %>` above would add all webpacker javascript, including autcomplete. Maybe we need to add `supplier-users` to `application.js`?

Reply:
I was trying to avoid loading it on every page 🤷

Reply:
Oh, my bad, didn't read it thoroughly enough. 

How big is the autocomplete js? Is it big enough that we want to justify the extra complexity of working with it introduced by only loading it when it is used?

Reply:
Looks to be about 61kb minified. I thought it was ~400kb, but that's the dev-server

Reply:
My personal opinion (after looking through some more mature project) is that our js is not really complex enough to selectively load it. And I would lean towards 61kb not being large enough to warrant its own load.

Reply:
Will move. I just feels _wrong_ somehow to have it loading and running on every page 🤷